pymemcache/test/test_client_hash.py

```diff
@@ -186,6 +186,8 @@ def test_no_servers_left_with_commands(self):
         )
 
         result = client.get('foo')
+        assert result is None
+        result = client.set('foo', 'bar')
         assert result is False
```

Comment:
Well I just added a test at the end of the one I changed.

```diff
diff --git a/pymemcache/test/test_client_hash.py b/pymemcache/test/test_client_hash.py
index 30b2130..1b2709f 100644
--- a/pymemcache/test/test_client_hash.py
+++ b/pymemcache/test/test_client_hash.py
@@ -187,6 +187,8 @@ class TestHashClient(ClientTestMixin, unittest.TestCase):

         result = client.get('foo')
         assert result is False
+        result = client.get_many(['foo'])
+        assert result == {}
```

The above will fail.